packages/execution-context/src/types/types.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * This interface contains the common attributes that we often want to know about
+ * in any execution within our domain.
+ */
+interface KnownMetadata {
```

Comment:
We shouldn't be including anything related to our domain since this is meant for public consumption.

Reply:
Good point, I'll remove this and we can add this to one of the more specialized packages.